docs/release-notes/highlights-0.9.0.asciidoc

```diff
@@ -0,0 +1,33 @@
+[[release-highlights-0.9.0]]
+== 0.9.0 Release Highlights
+
+[float]
+=== Kubernetes Distribution Support
+
+In addition to Google Kubernetes Engine (GKE) and vanilla Kubernetes, ECK now also works on
+Azure Kubernetes Service (AKS), Amazon Elastic Kubernetes Service (EKS) and
+Red Hat OpenShift 3.11+ (with some restrictions on APM server functionality)
+
+[float]
+=== APM Server
+
+ECK 0.9.0 brings support for a new custom resource that allows you to deploy APM server on Kubernetes.
+The ECK operator takes care of linking it up to an Elasticsearch output, if desired, as well.
+
+[float]
+=== User Provided HTTP Certificates
+
+In addition to self-signed certificates which ECK uses as a default, we now support user provided certificates for
+HTTP connections to Elasticsearch, Kibana or APM. Configure them via a regular k8s TLS secrets and
```

Comment:
Configure them with regular k8s TLS secrets

Reply:
just one, removing the 's'